packages/ember-views/tests/compat/attrs_proxy_test.js

```diff
@@ -1,108 +0,0 @@
-import View from 'ember-views/views/view';
-import { runAppend, runDestroy } from 'ember-runtime/tests/utils';
-import { compile } from 'ember-template-compiler';
-import run from 'ember-metal/run_loop';
-import { set } from 'ember-metal/property_set';
-import { get } from 'ember-metal/property_get';
-import { observer } from 'ember-metal/mixin';
-import { on } from 'ember-metal/events';
-import { registerKeyword, resetKeyword } from 'ember-htmlbars/tests/utils';
-import viewKeyword from 'ember-htmlbars/keywords/view';
-import buildOwner from 'container/tests/test-helpers/build-owner';
-import { OWNER } from 'container/owner';
-
-var view, owner, originalViewKeyword;
-
-import { test, testModule } from 'ember-glimmer/tests/utils/skip-if-glimmer';
-
-testModule('ember-views: attrs-proxy', {
-  setup() {
-    originalViewKeyword = registerKeyword('view',  viewKeyword);
-    owner = buildOwner();
-  },
-
-  teardown() {
-    runDestroy(view);
-    runDestroy(owner);
-    resetKeyword('view', originalViewKeyword);
-  }
-});
-
-test('works with properties setup in root of view', function() {
-  owner.register('view:foo', View.extend({
-    bar: 'qux',
-
-    template: compile('{{view.bar}}')
-  }));
-
-  view = View.extend({
-    [OWNER]: owner,
-    template: compile('{{view "foo" bar="baz"}}')
-  }).create();
-
-  runAppend(view);
-
-  equal(view.$().text(), 'baz', 'value specified in the template is used');
-});
-
-test('works with undefined attributes', function() {
-  // TODO: attrs
-  // expectDeprecation();
-
-  var childView;
-  owner.register('view:foo', View.extend({
-    init: function() {
-      this._super(...arguments);
-
-      childView = this;
-    },
-
-    template: compile('{{bar}}')
-  }));
-
-  view = View.extend({
-    [OWNER]: owner,
-    template: compile('{{view "foo" bar=undefined}}')
-  }).create();
-
-  runAppend(view);
-
-  equal(view.$().text(), '', 'precond - value is used');
-
-  run(function() {
-    set(childView, 'bar', 'stuff');
-  });
-
-  equal(get(view, 'bar'), undefined, 'value is updated upstream');
-});
-
-test('an observer on an attribute in the root of the component is fired when attrs are set', function() {
```

Comment:
This seems to be tested here https://github.com/emberjs/ember.js/blob/master/packages/ember-glimmer/tests/integration/components/curly-components-test.js#L1877